graphics/gpick/Portfile

```diff
@@ -2,12 +2,14 @@
 
 PortSystem              1.0
 PortGroup               boost 1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+PortGroup               lua 1.0
 
-name                    gpick
-version                 0.2.5
-revision                3
+github.setup            thezbyg gpick 0.3 v
+version                 0.3
```

Comment:
`github.setup` sets `version` for you.
```suggestion
```

Reply:
Thank you for noticing.
That line was a vestige from earlier testing.